resolveip: add page (#4982)
tldr-pages/tldr: pages/linux/resolveip.md

# resolveip

> > Resolve host name to IP address and vice versa.

Reviewer: I think hostname is a single word.
```suggestion
> Resolve hostnames to their IP addresses and vice versa.
```

> More information: <https://mariadb.com/kb/en/resolveip/>.

> - Resolve host name to IP address:

Reviewer: Perhaps this will read better?
```suggestion
- Resolve a hostname to an IP address:
```

`resolveip {{example.org}}`

> - Resolve IP address to host name:

Reviewer: Perhaps this will read better?
```suggestion
- Resolve an IP address to a hostname:
```
